backend/benefit/applications/api/v1/serializers.py

```diff
@@ -124,6 +128,80 @@ class Meta:
         }
 
 
+class EmployeeSerializer(serializers.ModelSerializer):
+    """
+    Employee objects are meant to be edited together with their Application object.
+    """
+
+    SSN_REGEX = r"^(\d{6})[aA+-](\d{3})([0-9A-FHJ-NPR-Ya-fhj-npr-y])$"
+
+    SSN_CHEKSUM = {
+        int(k): v
+        for k, v in xgroup(
+            """0    0    16    H
+                                                         1    1    17    J
+                                                         2    2    18    K
+                                                         3    3    19    L
+                                                         4    4    20    M
+                                                         5    5    21    N
+                                                         6    6    22    P
+                                                         7    7    23    R
+                                                         8    8    24    S
+                                                         9    9    25    T
+                                                         10    A    26    U
+                                                         11    B    27    V
+                                                         12    C    28    W
+                                                         13    D    29    X
+                                                         14    E    30    Y
+                                                         15    F""".split()
+        )
+    }
+
+    def validate_social_security_number(self, value):
```

Comment:
Perhaps this validation could use some comments / docs on how it works :grin:

Reply:
Added link to the specification
I lifted this code with small modifications from a previous project (where I have full rights to the code).